docs/serving/load-balancing/_index.md

```diff
@@ -0,0 +1,33 @@
+---
+title: "Load balancing"
+weight: 30
+type: "docs"
+---
+
+You can configure load balancing on Knative, by placing the _Activator service_ in the request path to act as a load balancer.
```

Comment:
I think it is more precise that you can `turn on **Knative** Load Balancing`. Since there's load balancing provided by k8s and/or mesh provider, if turned on, out of the box.